Traceback (most recent call last):
  File "/Users/runner/work/rfswarm/rfswarm/rfswarm_manager/rfswarm.py", line 6980, in <module>
    core = RFSwarmCore()
  File "/Users/runner/work/rfswarm/rfswarm/rfswarm_manager/rfswarm.py", line 2249, in __init__
    base.gui = RFSwarmGUI()
  File "/Users/runner/work/rfswarm/rfswarm/rfswarm_manager/rfswarm.py", line 3242, in __init__
    self.root = tk.Tk()
  File "/Users/runner/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/tkinter/__init__.py", line 2025, in __init__
    self._loadtk()
  File "/Users/runner/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/tkinter/__init__.py", line 2040, in _loadtk
    % (_tkinter.TK_VERSION, tk_version))
RuntimeError: tk.h version (8.5) doesn't match libtk.a version (8.6)
